active-rfcs/0000-custom-elements-interop.md

```diff
@@ -28,8 +31,8 @@ The default generated render function code is (pseudo code):
 
 ```js
 function render() {
-  const component_plastic_button = resolveComponent('plastic-button')
-  return createVNode(component_plastic_button)
+  const component_plastic_button = resolveComponent("plastic-button");
```

Comment:
Code style seems inconsistent with the later part of the RFC, maybe something wrong with the formatter?

Reply:
We could add a `.prettierrc` file to make it automatic for people using a plugin with their IDE

Reply:
done